ERCS/eip-xxxx.md

```diff
@@ -0,0 +1,266 @@
+---
+title: dApp Permission Framework (DPF)
+description: Declarative permissions to minimize impact when a web dApp is compromised.
+author: Justin Phu (@jqphu)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: Interface
+created: 2024-02-12
+requires: [EIP712](../EIPS/eip-712.md]
+---
+
+## Abstract
+
+There have been many instances of official web dApps being compromised. There are a plethora of ways this can happen such as a supply chain attack, a malicious code update, DNS compromises (e.g. DNS credential compromise or BGP attack).
+
+In this proposal, we propose a mechanism for dApps to declare what interactions they allow which will vastly reduce the efficacy of these attacks and in some situations prevent them entirely.
+
+![[../assets/eip-xxxx/dApp_permission_framework_high_level.png]]
```

Comment:
Your abstract could stand to have a bit more technical detail. This section should give the reader a brief/high-level (but still technical) overview of your proposal.